source/setup.py

```diff
@@ -190,7 +190,15 @@ def getRecursiveDataFiles(dest,source,excludes=()):
 	options = {"py2exe": {
 		"bundle_files": 3,
 		"excludes": ["tkinter",
-			"serial.loopback_connection", "serial.rfc2217", "serial.serialcli", "serial.serialjava", "serial.serialposix", "serial.socket_connection"],
+			"serial.loopback_connection", 
+			"serial.rfc2217", 
+			"serial.serialcli", 
+			"serial.serialjava", 
+			"serial.serialposix", 
+			"serial.socket_connection",
+			# Py2exe seems to implicitly include this, but we don't need it. 
+			"win32api",

```

Comment:
I'd really like to know why this was included.

Also note the following lines from the output:

```
Copy d:\Development\NVDA\source repo\include\py2exe\win32wnet.pyd to D:\Development\NVDA\source repo\dist\win32wnet.pyd
Copy d:\Development\NVDA\source repo\include\py2exe\_winxptheme.pyd to D:\Development\NVDA\source repo\dist\_winxptheme.pyd
Copy ExtensionDLL d:\Development\NVDA\source repo\include\py2exe\pywintypes37.dll to D:\Development\NVDA\source repo\dist\
```

These 3 files are all pywin32. We also have the following files in library.zip:

* win32wnet.pyc
* winxptheme.pyc
* _winxptheme.pyc